packages/pocs/pact-project-proposal/chainweb-pact-project.json

```diff
@@ -0,0 +1,18 @@
+{
+  "name": "chainweb-pact",
+	"description": "Chainweb Pact Project",
+  "source": "git@github.com:kadena-io/chainweb-pact.git",
+	"dependencies": [
+		"git@github.com:other-user/my-other-dependency.git"
```

Comment:
I like the idea of content address-ability here. We should figure out how to reference contract hashes directly with some nice syntax. Having github support would be killer to mix with that.

Reply:
I'd like this to be similar to how NPM allows to use dependencies. This includes various ways of specifying versions, git urls, github urls (`kadena-community/a-repo#commit-sh`) etc

https://docs.npmjs.com/cli/v10/configuring-npm/package-json#dependencies

Reply:
That's fine by me, but we should specify the variations in syntax that we plan to support beforehand.